StickyHeaders/src/main/java/io/doist/recyclerviewext/sticky_headers/StickyHeadersLinearLayoutManager.java

```diff
@@ -603,4 +698,43 @@ private void sortHeaderAtIndex(int index) {
             }
         }
     }
+
+    public static class SavedState implements Parcelable {
+        private Parcelable superState;
+        private int pendingScrollPosition;
+        private int pendingScrollOffset;
+
+        public SavedState() {
+        }
+
+        public SavedState(Parcel in) {
+            superState = in.readParcelable(StickyHeadersLinearLayoutManager.class.getClassLoader());
```

Comment:
:+1: Yeah, both should work, but it's more intuitive to use `SavedState`'s. Fixed in 7b55eeb1a8dd61b06590f80e3ae724e830894afb